build

```diff
@@ -29,6 +31,15 @@ while :; do
         -c|--use-cache)
             CACHE_FLAG=''
             ;;
+        -b|--base-image-tag)
+            if [[ -z $2 ]]; then
```

Comment:
Using `"$2"` would allow spaces in `$2`, however unlikely.

Reply:
Docker image tags can't have spaces. I can add the `""` but it would fail at `docker build`.

Reply:
I would pick the option that makes the Docker limitation the most clear.  E.g. if tag `"with space"` results in an error saying "Docker image tags can't have spaces", I think that would be preferable.  But, leaving it to you if it's worth the additional investigation & testing.